app/lang/en.json

```diff
@@ -23,7 +23,9 @@
     "selectOneOrMoreOptions": "Select one or more options.",
     "answerLater": "Answer later",
     "willDoThisLater": "I will do this later",
-    "pfa": "Finances and affairs",
```

Comment:
I double checked, we're no longer using this trans key since implementing the humanised version of it